core/imageroot/usr/local/agent/pypkg/agent/backup/__init__.py

```diff
@@ -33,6 +35,7 @@ class Restic:
     rootfull = False
     restic_dir = ""
     restic_env = ""
+    restic_image = "ghcr.io/nethserver/restic"
```

Comment:
The value should be read from core.env. For example:

```python
import agent
core_env = agent.read_envfile('/etc/nethserver/core.env')
print(core_env["RESTIC_IMAGE"])
```

Reply:
I prefer to leave it inside the library

Reply:
So, why did you add it to core.env? An hardcoded value would be enough.

Setting the value in core.env allows to run `podman pull $RESTIC_IMAGE` during modules installation (and just before the first backup run).

Reply:
Because you asked to add it to the env :)